scripts/tts_comparison_report/reporting/metrics/__init__.py

```
@@ -0,0 +1,22 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from scripts.tts_comparison_report.reporting.metrics.registry import DistributionMetricsRegistry, MetricsRegistry
+from scripts.tts_comparison_report.reporting.metrics.specs import DistributionMetricSpec, MetricSpec
+
+__all__ = [
+    "DistributionMetricsRegistry",
+    "MetricsRegistry",
+    "DistributionMetricSpec",
+    "MetricSpec",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)